src/app/pages/SingleEntry/SingleEntry.tsx

```diff
@@ -63,7 +63,16 @@ function SingleEntry(): JSX.Element {
           <img className={styles.image} src={entry.imageUrl} alt="" />
           <h1>{entry.title}</h1>
           <p className={styles.text}>{entry.text}</p>
-          <Button name="delete" type="button" onClick={() => handleClick()} />
+          <span className={styles.buttonContainer}>
+            <Link
+              key={entry.date}
+              to={`/edit/${entry.date}`}
+              className={styles.edit}
+            >
+              Edit
+            </Link>
+            <Button name="Delete" type="button" onClick={() => handleClick()} />
```

Comment:
I guess the name is used for the text of the button, is it? I would suggest:
- (if you haven't done it yet) giving the name in your button component the type:
`name: 'delete' | //other things you use your button for`
- writing 'delete' in lowercase
- and then use CSS to type the first letter of the button text in uppercase

Reply:
👍